pandas/tests/scalar/timestamp/test_timestamp.py

```diff
@@ -223,6 +227,31 @@ def test_resolution(self):
         assert dt.as_unit("ms").resolution == Timedelta(milliseconds=1)
         assert dt.as_unit("s").resolution == Timedelta(seconds=1)
 
+    @pytest.mark.parametrize(
+        "date_string, expected_true, expected_false",
+        [
+            ("0000-2-29", 1, 2),
+            ("0000-3-1", 2, 3),
+            ("1582-10-14", 3, 4),
+            ("1582-10-15", 4, 5),
+            ("1752-01-01", 5, 6),
+            ("2023-06-18", 6, 0),
+        ],
+    )
+    def test_dow_historic(self, date_string, expected_true, expected_false):
+        # GH 53738
+        dt = Timestamp(date_string)
```

Comment:
`dt` is often used as an alias for the stdlib `datetime` (`import datetime as dt`) - can we use `ts` instead (for "timestamp")? same in the test below